mmv1/third_party/terraform/website/docs/r/logging_project_sink.html.markdown

```diff
@@ -89,6 +89,47 @@ resource "google_project_iam_binding" "gcs-bucket-writer" {
 }
 ```
 
+The following example creates a sink that are configured with user-managed service accounts, by specifying
```

Comment:
Now that there are 4 examples here, it could make sense to add headers for each example so that they show up in the sidebar. See https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/alloydb_cluster for comparison.

Reply:
To clarify, I just need to add extra headers in the markdown page as following, they will automatically show up in the sidebar? 
```
## Example Usage - Basic sink
```

Reply:
yep

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/r/bigtable_instance.html.markdown is a similar handwritten doc that does this - rendered as https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigtable_instance

Reply:
Updated.